Makefile

```
@@ -35,9 +37,22 @@ luacheck:
 
 .PHONY: test
 test:
-	luatest -v --shuffle all:${SEED}
+	luatest -v --coverage --shuffle all:${SEED}
```

The `coverage` slows the work of tests, their results may differ. It's better to do two different launches.
```suggestion
	luatest -v --shuffle all:${SEED}

test-coverage:
	luatest -v --coverage --shuffle all:${SEED}
```